shared/constants/misc.js

```diff
@@ -86,6 +86,14 @@ const dietaryRestrictions = [
     'No soybeans',
 ]
 
+const hearAboutEventOptions = [
+    'Facebook',
+    'Twitter',
+    'Instagram',
+    'Other social media',
+    'Friend',
+]
```

Comment:
Do we want more options than this?

Reply:
This one probably needs some changing right? We should probably try to get this solved